pom.xml

```diff
@@ -13,7 +13,7 @@
 
```

Comment:
There are a handful of references to the AllenMLI org in this file that probably should be updated.

Reply:
Good catch